### ISSUE TITLE
ImportError: cannot import name 'Markup' from 'jinja2'

### ISSUE THREAD

**Issue by russellvt** [russellvt/hockey-info#1]
Fresh install using `Python 3.9.16`, `Flask 1.0.2` and `Jinja2 3.1.2`.

Complains that Markup is not in jinja2.

```
(venv-hockey) russell@host:~/projects/hockey/hockey-info$ flask run --host=0.0.0.0
Traceback (most recent call last):
  File "~/projects/hockey/hockey-info/venv-hockey/bin/flask", line 5, in <module>
    from flask.cli import main
  File "~/projects/hockey/hockey-info/venv-hockey/lib/python3.9/site-packages/flask/__init__.py", line 19, in <module>
    from jinja2 import Markup, escape
ImportError: cannot import name 'Markup' from 'jinja2' (~/projects/hockey/hockey-info/venv-hockey/lib/python3.9/site-packages/jinja2/__init__.py)
(venv-hockey) russell@host:~/projects/hockey/hockey-info$ python --version
Python 3.9.16
(venv-hockey) russell@host:~/projects/hockey/hockey-info$ pip freeze
arrow==0.13.1
certifi==2022.12.7
chardet==3.0.4
click==8.1.3
Flask==1.0.2
gunicorn==20.1.0
idna==2.8
itsdangerous==2.1.2
Jinja2==3.1.2
MarkupSafe==2.1.2
python-dateutil==2.8.2
requests==2.21.0
requests-cache==0.4.13
six==1.16.0
urllib3==1.24.3
Werkzeug==2.3.2
```

**Comment by russellvt** [comment]
This is a "feature" of newer Jinja2 versions. Now we need:

    from jinja2.utils import markupsafe 
    markupsafe.Markup()
    Markup('')

Ref:
 - https://stackoverflow.com/questions/71645272/importerror-cannot-import-name-markup-from-jinja2
 - https://itsourcecode.com/python-tutorial/cannot-import-name-markup-from-jinja2-solved/

**Comment by russellvt** [comment]
Really, we should just upgrade Flask to v2 or beyond.

**Comment by russellvt** [comment]
Fixed in Merge Commit bd1d3ef779a1c29107104a40d9592582596cf214
